code/controllers/subsystems/ticker.dm

```diff
@@ -438,6 +438,23 @@ var/datum/controller/subsystem/ticker/SSticker
 
 	to_world("<B><span class='notice'>Welcome to the pre-game lobby!</span></B>")
 	to_world("Please, setup your character and select ready. Game will start in [pregame_timeleft] seconds.")
+
+	// Compute and, if available, print the ghost roles in the pre-round lobby. Begone, people who do not ready up to see what ghost roles will be available!
+	var/list/available_ghostroles = list()
+
+	for(var/s in SSghostroles.spawners)
+		var/datum/ghostspawner/G = SSghostroles.spawners[s]
+		if(G.enabled && !("Antagonist" in G.tags) && !(G.loc_type == GS_LOC_ATOM && !length(G.spawn_atoms)))
+			available_ghostroles |= list(G.name)
+
+	// Special case, to list the Merchant in case it is available at roundstart
+	if(SSjobs.type_occupations[/datum/job/merchant].total_positions)
+		available_ghostroles |= list(SSjobs.type_occupations[/datum/job/merchant].title)
+
+	if(length(available_ghostroles))
+		to_world("<br /><br /><B><span class='notice'>Ghost roles available for this round:</span></B> [english_list(available_ghostroles)].<br />\
+		<span class='info'>Please note that the actual availability depends on additional things, including your user (eg. job bans)</span>")
```

Comment:
Would prefer if you used the SPAN_INFO / SPAN_NOTICE / SPAN_BOLD macros here.

Reply:
This entire file uses the raw <span> tags. Would be inconsistent to do it to just this section. I recommend doing so for the entire file personally.

Reply:
Ideally yes, but I am not going to ask them to go through everything. Using the SPAN macros for new stuff is enough.